drivers/display/display_stm32_ltdc.c

```diff
@@ -250,11 +250,7 @@ static int stm32_ltdc_read(const struct device *dev, const uint16_t x,
 static int stm32_ltdc_display_blanking_off(const struct device *dev)
 {
 	const struct display_stm32_ltdc_config *config = dev->config;
-	const struct device *display_dev = config->display_controller;
-
-	if (display_dev == NULL) {
-		return 0;
-	}
+	const struct device *display_dev = config->panel_controller;
 
 	if (!device_is_ready(display_dev)) {
 		LOG_ERR("Display device %s not ready", display_dev->name);
```

Comment:
Same remark.

Same nit:
```suggestion
		LOG_ERR("Panel device %s not ready", display_dev->name);
```